src/fe/implicit/_.js

```diff
@@ -34,26 +41,81 @@ export function validateArgs(...validators) {
     };
 }
 
+/**
+ * Matches the type of the first argument with the second argument using typeof operator and returns true or false
+ * @param {*} x
+ * @param {string} type
+ *
+ * @returns {boolean}
+ */
 export const isType = ((x, type) => typeof x === type) |> curry2;
 export const isBoolean = isType('boolean');
 export const isNumber = isType('number');
 export const isString = isType('string');
 export const isFunction = isType('function');
 export const isObject = isType('object');
 export const isArray = Array.isArray;
+
+/**
+ * Checks if the given argument is an element or not
+ * @param {Object | Element} o
+ *
+ * @returns {boolean}
+ */
 export const isElement = o => isNonNullObject(o) && o.nodeType === 1;
-export const isTruthy = o => o;
 
+/**
+ * Checks if the given argument is truthy or not
+ * @param {*} o
+ *
+ * @returns {boolean}
+ */
+export const isTruthy = o => Boolean(o);
+
+/**
+ * Checks if the given argument is not a null object
+ * @param {Object} o
+ *
+ * @returns {boolean}
+ */
 export const isNonNullObject = o => o !== null && isObject(o);
+
+/**
+ * Checks if the given object is empty or not
+ * @param {Object} x
+ *
+ * @returns {boolean}
+ */
 export const isEmptyObject = x => !lengthOf(Object.keys(x));
 
-// create getProperty function based on keys
+/**
+ * Create getProperty function based on keys
+ * @param {object} obj
+ * @param {string} key
+ *
+ * @returns {*}
+ */
 export const prop = curry2((obj, key) => obj && obj[key]);
 
 export const lengthOf = prop('length');
 export const prototypeOf = prop('prototype');
 
+/**
+ * Checks if the constructor of first parameter is same as second parameter constructor
+ * @param {*} x
+ * @param {function} y
+ *
+ * @returns {boolean}
+ */
 export const isExact = curry2((x, y) => x && x.constructor === y);
+
+/**
+ * Checks if the first parameter is an instance of second parameter class
+ * @param {*} x
+ * @param {class} y
```

Comment:
If `{class}` does not throw any JSDoc linting errors, use it in the second param of `isExact` as well.